charts/gateway/templates/_helpers.tpl

```diff
@@ -193,6 +193,17 @@ Define OTK Image Pull Secret Name
 {{- end -}}
 {{- end -}}
 
+{{/*
+ Define Gateway node.properties Secret Name
+ */}}
+{{- define "gateway.node.properties" -}}
```

Comment:
Don't know if this is a problem, but `.` is a special character in yaml, so this would probably be safer to call `gateway.nodeproperties`

Reply:
This is only a defined variable , but we can change it , I just wanted to node.properties at all places